tests/nvme_test.py

```diff
@@ -0,0 +1,156 @@
+import unittest
+import os
+import overrides_hack
+
+from utils import create_sparse_tempfile, create_nvmet_device, delete_nvmet_device, fake_utils, fake_path, run_command, run, TestTags, tag_test
+from gi.repository import BlockDev, GLib
+from distutils.spawn import find_executable
+
+
+class NVMeTest(unittest.TestCase):
+    requested_plugins = BlockDev.plugin_specs_from_names(("nvme",))
+
+    @classmethod
+    def setUpClass(cls):
+        if not BlockDev.is_initialized():
+            BlockDev.init(cls.requested_plugins, None)
+        else:
+            BlockDev.reinit(cls.requested_plugins, True, None)
+
+
+class NVMeTestCase(NVMeTest):
+    def setUp(self):
+        if not find_executable("nvme"):
+            raise unittest.SkipTest("nvme executable (nvme-cli package) not found in $PATH, skipping.")
+        if not find_executable("nvmetcli"):
```

Comment:
Please add both `nvme-cli` and `nvmetcli` to the [install-test-dependencies](https://github.com/storaged-project/libblockdev/blob/master/misc/install-test-dependencies.yml) playbook.

Reply:
Done, good idea. Debian doesn't ship `nvmetcli`, tests won't run.